way_to_home/static/src/components/userSettingsForm/notifications/notificationForm.js

```diff
@@ -0,0 +1,289 @@
+import React, {Component} from 'react';
+import axios from 'axios';
+import Button from "@material-ui/core/Button";
+import CustomizedSnackbars from '../../message/message';
+import NotificationItem from './notificationItem/notificationItem';
+import NewNotificationItem from './newNotificationItem/newNotificationItem'
+import './notificationForm.css';
+import Calendar from "react-calendar";
+import Chip from "@material-ui/core/Chip/Chip";
+import TrendingFlat from '@material-ui/icons/TrendingFlat';
+
+
+class NotificationForm extends Component{
+
+    state = {
+        pointA: String(undefined),
+        pointB: String(undefined),
+        OpenStartDate: false,
+        OpenEndDate: false,
```

Comment:
edit variables using js conversation. For example: `variableName` instead of 'VariableName'

Reply:
fixed